sentry-sidekiq/spec/spec_helper.rb

```diff
@@ -133,8 +133,10 @@ def perform
   end
 end
 
+class HappyWorkerForCron < HappyWorker; end
```

Comment:
Why do we need another `HappyWorkerForCron`?

Reply:
to keep each class isolated after patching, since we don't have a 'reverse prepend

Reply:
No I mean we already had this worker before this PR, so now we have 2:

https://github.com/getsentry/sentry-ruby/blob/9b24de9d0a9019d99e2db56d7c5c05d0a8c16e99/sentry-sidekiq/spec/spec_helper.rb#L136-L137

Reply:
ah hmm that was probably copypasta error, thx